app/database/operator/server_data_operator/handlers/channel.test.ts

```diff
@@ -201,6 +203,72 @@ describe('*** Operator: Channel Handlers tests ***', () => {
         });
     });
 
+    it('=> HandleMyChannel: should handle is_unread and is_muted', async () => {
```

Comment:
Not sure if this was the right place for this but I wanted to test the changes somewhere.